Userland/Libraries/LibCrypto/Curves/EllipticCurve.h

```diff
@@ -17,6 +17,7 @@ class EllipticCurve {
     virtual ErrorOr<ByteBuffer> generate_public_key(ReadonlyBytes a) = 0;
     virtual ErrorOr<ByteBuffer> compute_coordinate(ReadonlyBytes scalar_bytes, ReadonlyBytes point_bytes) = 0;
     virtual ErrorOr<ByteBuffer> derive_premaster_key(ReadonlyBytes shared_point) = 0;
+    virtual ErrorOr<bool> check_signature(ReadonlyBytes hash, ReadonlyBytes pubkey, ReadonlyBytes signature) = 0;
```

Comment:
Imo this should be named `verify` instead, to closer match the TLS spec language of sign/verify

Reply:
Sure